src/gsy_e/models/strategy/state/storage_state.py

```diff
@@ -43,16 +49,28 @@ class ESSEnergyOrigin(Enum):
 EnergyOrigin = namedtuple("EnergyOrigin", ("origin", "value"))
 
 
+@dataclass
+class StorageLosses:
+    """Container for all loss related Storage settings."""
+
+    charging_loss_percent: float = 0.0
+    discharging_loss_percent: float = 0.0
+    self_discharge_per_day_kWh: float = 0.0
+
+
 # pylint: disable= too-many-instance-attributes, too-many-arguments, too-many-public-methods
 class StorageState(StateInterface):
     """State for the storage asset."""
 
-    def __init__(self,
-                 initial_soc=StorageSettings.MIN_ALLOWED_SOC,
-                 initial_energy_origin=ESSEnergyOrigin.EXTERNAL,
-                 capacity=StorageSettings.CAPACITY,
-                 max_abs_battery_power_kW=StorageSettings.MAX_ABS_POWER,
-                 min_allowed_soc=StorageSettings.MIN_ALLOWED_SOC):
+    def __init__(
+        self,
+        initial_soc=StorageSettings.MIN_ALLOWED_SOC,
+        initial_energy_origin=ESSEnergyOrigin.EXTERNAL,
+        capacity=StorageSettings.CAPACITY,
+        max_abs_battery_power_kW=StorageSettings.MAX_ABS_POWER,
+        min_allowed_soc=StorageSettings.MIN_ALLOWED_SOC,
+        losses: Optional[StorageLosses] = StorageLosses(),
+    ):
```

Comment:
Minor, but since it is `Optional` I would prefer `None` as default value:
```suggestion
        losses: Optional[StorageLosses] = None,
    ):
        if not losses:
            losses = StorageLosses()
```